README.md

```diff
@@ -49,6 +49,7 @@ The KubeRay repository only contains documentation related to the development an
 * [Profiling with py-spy](https://docs.ray.io/en/master/cluster/kubernetes/k8s-ecosystem/pyspy.html#kuberay-pyspy-integration)
 * [KubeRay integration with Volcano](https://docs.ray.io/en/master/cluster/kubernetes/k8s-ecosystem/volcano.html#kuberay-volcano)
 * [Kubeflow: an interactive development solution](https://docs.ray.io/en/master/cluster/kubernetes/k8s-ecosystem/kubeflow.html#kuberay-kubeflow-integration)
+* [KubeRay integration with MCAD (Multi-Cluster-App-Dispatcher)](https://github.com/ray-project/kuberay/blob/master/docs/guidance/kuberay-with-MCAD.md)
```

Comment:
Users who aren't familiar with MCAD might not understand what's the purpose of the doc. Maybe use something like "MCAD: An out-of-box Kubernetes Solution for Gang Scheduling and Multi-Cluster Job Dispatching". I am not familiar with MCAD, so you may be able to come up with a better title.

Reply:
How about this ?
MCAD: A Kubernetes controller for Queuing, Gang Scheduling jobs on single or multi-cluster environment.